eng/common/scripts/artifact-metadata-parsing.ps1

```diff
@@ -410,10 +410,16 @@ function VerifyPackages($pkgRepository, $artifactLocation, $workingDirectory, $a
         exit(1)
       }
 
+      $tag = if ($parsedPackage.packageId) {
+        "$($parsedPackage.packageId)_$($parsedPackage.PackageVersion)"
+      } else {
+        $parsedPackage.PackageVersion
+      }
+
       $pkgList += New-Object PSObject -Property @{
         PackageId      = $parsedPackage.PackageId
         PackageVersion = $parsedPackage.PackageVersion
```

Comment:
I assume you aware but just to call out that changes under eng/common will eventually need to be made in the azure-sdk-tools repo before merging in the language repo. Adding them in the language repo first for testing is definitely a good strategy but just need to ensure correct ordering when you are ready to merge.

Reply:
https://github.com/Azure/azure-sdk-tools/pull/644